config/sync/system.performance.yml

```diff
@@ -2,15 +2,15 @@ cache:
   page:
     max_age: 0
 css:
-  preprocess: true
+  preprocess: false
```

Comment:
Same issue as here: https://github.com/Pronovix/food/pull/17#discussion_r398096150

Otherwise it looks good :+1: